src/scoping-shim.js

```diff
@@ -79,7 +79,7 @@ export default class ScopingShim {
    * @param {string=} typeExtension
    */
   prepareTemplateStyles(template, elementName, typeExtension) {
-    if (template._prepared) {
+    if (template._prepared || disableRuntime) {
```

Comment:
should this also be in `prepareTemplateDom`?

Reply:
Will do

Reply:
Done